string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,15 @@
+/* exported capitalizeWord */
+// check if given word is 'javascript' word
+// before do all charters in lower or upper case
+// in if condition check if word is strikly equal to 'javascript' string
+// if yes, do first charater uppercase concatenate with next sectiom in lower case
+// and for charater at index 4 use touppercase again
+// if condition is false do next step - first letter to upper, rest lower
+function capitalizeWord(word) {
+  if (word.toUpperCase() === 'JAVASCRIPT') {
+    return word.charAt(0).toUpperCase() + word.slice(1, 4).toLowerCase() + word.charAt(4).toUpperCase() + word.slice(5).toLowerCase();
```

Comment:
In special cases like this, you don't need to manually put this together. You can just `return "javaScript";`